core/src/main/java/fr/sncf/osrd/exceptions/ErrorContext.java

```diff
@@ -1,13 +1,15 @@
 package fr.sncf.osrd.exceptions;
 
 import com.squareup.moshi.adapters.PolymorphicJsonAdapterFactory;
+import fr.sncf.osrd.new_infra_state.implementation.SignalizationError;
 
 public class ErrorContext {
 
     public static final PolymorphicJsonAdapterFactory<ErrorContext> adapter = (
             PolymorphicJsonAdapterFactory.of(ErrorContext.class, "type")
                     .withSubtype(Train.class, "train")
                     .withSubtype(Allowance.class, "allowance")
+                    .withSubtype(SignalizationError.SignalContext.class, "signal")
```

Comment:
could you add the error context in this file as well, for the sake of consistency?